examples/DotNetCore/AzureFunction/FunctionApp/Function1.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Logging;
+
+namespace FunctionApp
+{
+    public static class Function1
+    {
+        static readonly MyConfig _myConfig = new MyConfig();
+
+        [FunctionName("Function1")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            log.LogInformation("C# HTTP trigger function processed a request.");
+
+            // Configuration will be refreshed if the cache is expired and the 'Sentinel' key is modified.
+            // Remove the 'await' operator if the configuration is preferred to be refreshed without blocking.
+            await _myConfig.ConfigurationRefresher.Refresh();
+
+            string keyName = "TestApp:Settings:Message";
+            string message = _myConfig.Configuration[keyName];
+            
+            return message != null
+                ? (ActionResult)new OkObjectResult(message)
+                : new BadRequestObjectResult($"Please pass a message from App Configuration with key name '{keyName}'");
```

Comment:
>pass a message from App Configuration with key name [](start = 54, length = 51)

The code is already set up to pass the message. IF they are seeing this then I would expect they just forgot to create the key-value in the app configuration. We could be more explicit and say "Please create a key-value with the key '{keyName}' in App Configuration."

Reply:
Sounds good.